test/test.js

```diff
@@ -14,7 +14,8 @@ require('../glpk.js').then(glpkjs => {
 
 			let json = JSON.parse(fs.readFileSync(`${__dirname}/data/${d}.json`).toString());
 			let prob = new glpk.Problem();
-			let z1, z2 = glpkjs.solve(json, glpkjs.GLP_MSG_ERR).result.z;
+			const settings = json.settings
```

Comment:
Should we add a test to test that the settings work as expected? Maybe we can a medium sized problem and set the tmLim to 0 and see if it works.

Reply:
It's a good idea but since I have removed the building process, it means the changes haven't taken place. So in another PR after we build the changes, we can create new tests.

Reply:
You are totally right